### PR TITLE
feat: add color and tag to note

### PR DESCRIPTION
**Pull request by yaswanthmyneni**
# Add color and tag to note

## Preview: [Lucent Notes](https://deploy-preview-6--lucent-notes-dev.netlify.app/)

recording

### CODE REVIEW

File: src/components/color-pallette-modal/ColorPalletteModal.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      type: "SET_COLOR",
+      payload: event.currentTarget.className.split(" ")[1],
+    });
+    setIsColorsModal(false);
+  };
+
+  return (
+    <div className="color-modal">
+      {colors.map((className) => (
+        <div
```
Comment (Ankur9669) [inline src/components/color-pallette-modal/ColorPalletteModal.jsx:30]: Use key here

File: src/components/note-modal/NoteModal.jsx [hunk 2]
```diff
@@ -1,69 +1,143 @@
-import { AiFillCloseCircle } from "assets/icons/icons";
+import { useState } from "react";
+import { AiFillCloseCircle, IoColorPaletteOutline } from "assets/icons/icons";
 import { useNotesContext, useToastContext } from "context";
 import { addToNotes, editNote } from "utility";
+import { ColorPalletteModal } from "components";
 import "./NoteModal.css";
```
Comment (Ankur9669) [inline src/components/note-modal/NoteModal.jsx:7]: Can import all files in one single file

**Review by Ankur9669 — APPROVED** [review]
The code looks fine to me you can merge it after looking to the suggestions

File: src/components/note-modal/NoteModal.jsx [hunk 3]
```diff
@@ -1,69 +1,143 @@
...
+                name="label"
+                id="food"
+                checked={labelForNote === "food"}
+                onChange={() =>
+                  notesDispatch({ type: "LABEL_FOR_NOTE", payload: "food" })
+                }
+              />
+              Food
+            </label>
+            <label htmlFor="home" className="cursor">
```
Comment (nikhiljugale007) [inline src/components/note-modal/NoteModal.jsx]: suggestion: You can move this tag code to separate components so you can reuse them without repeating code if number of tag increases

**Review by nikhiljugale007 — COMMENTED** [review]
Good to merge. 
Followed good naming convention.

**Pull request merged**